src/dotnet/APIView/APIViewWeb/ExtensionMethods.cs

```diff
@@ -9,4 +12,41 @@ public static string GetGitHubLogin(this ClaimsPrincipal user)
             return user.FindFirst(c => c.Type == "urn:github:login")?.Value;
         }
     }
+    public class OrganizationRequirement: IAuthorizationRequirement
+    {
+        public string Org { get; set; }
+        public OrganizationRequirement(string org)
+        {
+            Org = org;
+        }
+    }
+
+    public class OrganizationRequirementHandler : IAuthorizationHandler
+    {
+        public Task HandleAsync(AuthorizationHandlerContext context)
+        {
+            foreach (var r in context.Requirements)
+            {
+                if (r is OrganizationRequirement o)
+                {
+                    try
+                    {
+                        if (context.User.FindFirst("urn:github:orgs").Value.Split(",").Contains(o.Org))
```

Comment:
Would FindFirst return null if a claim doesn't exist?

Reply:
It does - I'm not sure if there's a better way to go about this check. I added a requirement in the `RequireOrganizationRequirement` for an organizations claim, but when the site initially spins up, this line in the code is triggered and returns null.

Reply:
Check claim for null first